Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream> 
+#include "Person.h"
+
+
+
+
+std::string(n);
+int a;
+
+Person::Person() 
+{
+
+}
+
+Person::~Person() 
+{
+
+}
```

Comment:
Hint: something goes in these functions. 
Person() is a constructor so something like `this.name = "name" ` should be in there. 
~Person() is a destructor so it is used to delete a pre-allocated variable. Hint: it might be age.